etc/adminhtml/system.xml

```diff
@@ -1,21 +1,21 @@
 <?xml version="1.0"?>
-<!-- 
+<!--
 /**
 * Copyright © 2015 PlazaThemes.com. All rights reserved.
 
 * @author PlazaThemes Team <contact@plazathemes.com>
 */
 -->
-<config 
+<config
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../../Config/etc/system_file.xsd">

```

Comment:
Can you please change this to `xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Config:etc/system_file.xsd"`?